acceptance/README.md

```diff
@@ -116,10 +114,6 @@ in your hash; the environment variable names are the same but uppercased
   from source. This is primarily so we can test against alternate versions of
   Puppet, so if the Puppet repo is not specified we fall back to using packages.
 
-* `:puppetdb_repo_puppetdb` (`PUPPETDB_REPO_PUPPETDB`) :
```

Comment:
It now does a local tar.gz build, transfers that to the remote host, then unpacks it there etc. it not longer needs to have the repo, as in no longer needs to do a secondary pull of that repo. It uses the local repo for all its needs.